free-programming-books.md

```diff
@@ -360,8 +360,10 @@
 
 #### Computer Science
 
+* [A New Kind of Science](http://www.wolframscience.com/nks/) - Stephen Wolfram
```

Comment:
I think this is a duplicate- under cellular autonomata

Reply:
yes,  it's a duplicate